#goal
float32 turnDegree
---
#result
float32 turnedDegree
---
#feedback
float32 remainDegree
